README.md

```diff
@@ -73,3 +73,18 @@ utils/bin/firrtl -i regress/rocket.fir -o regress/rocket.v -X verilog // Compile
 utils/bin/firrtl --help // Returns usage string
 ```
 
+##### Citing Firrtl
+
+If you use Firrtl in a paper, please cite the following technical report, located here:
```

Comment:
"following technical report, located here:" repetitive.
How about:
"please cite the technical report, located here:"
or just:
"please cite the following technical report:"